packages/components/src/DataList/components/DataListHeaderTile/components/DataListSortingOptions.css

```diff
@@ -0,0 +1,44 @@
+.optionsList {
+  position: absolute;
+  top: 100%;
+  left: 0;
+  max-width: var(--popover--width);
+  box-shadow: var(--shadow-base);
+  margin-top: var(--space-smaller);
+  padding: var(--space-small) 0;
+  border: var(--border-base) solid var(--color-border);
+  border-radius: var(--radius-larger);
+  background: var(--color-surface);
+  align-items: flex-start;
+}
+
+.option {
+  display: flex;
+  min-width: calc(var(--space-largest) * 3.5);
+  min-height: calc((var(--space-largest) - var(--space-smaller)));
+  box-sizing: border-box;
+  margin: 0 var(--space-small);
```

Comment:
I've seen this in a few places now where we're specifying left and right margins on the options, and only top/bottom padding on the container. Is there a reason we're not just providing padding all around on the container?

Current
![image](https://github.com/GetJobber/atlantis/assets/39704901/dbbc5874-e319-497a-93c9-97751c6607e2)
![image](https://github.com/GetJobber/atlantis/assets/39704901/4734bd71-fb0d-466a-a8c8-129f7f2e214d)

Just using padding on the parent
![image](https://github.com/GetJobber/atlantis/assets/39704901/0e80555f-34d3-4550-9187-fefc96dcaf79)